packages/rancher-monitoring/prometheus-federator/package.yaml

```diff
@@ -1,4 +1,4 @@
 url: https://github.com/rancher/prometheus-federator.git
 subdirectory: charts/prometheus-federator/0.1.1
 commit: aa0be6fdb3aa13098e3c5a2051ff664fea5726d8
-version: 0.1.1
+version: 1.0.0
```

Comment:
Any reason to start version from 1.0.0 than 0.2.0?

Reply:
@prachidamle two reasons:
1. This would follow the versioning guidelines outlined in https://github.com/rancher/charts#versioning-charts
2. Bumping the major version gives us more flexibility since we have two versions we can modify; let's say for example that Helm Project Operator breaks on 1.25 clusters and has two upstream versions; one that works on 1.16-1.24 and one that works on 1.18-1.25. In that case, we might want to have two release lines, which is only possible if we leave the minor version untouched.

Reply:
makes sense, thanks